CHANGES.md

```diff
@@ -0,0 +1,14 @@
+# Changes made from Labrador master branch #
```

Comment:
Now that this is being merged back, could make more sense to rewrite this as a more general `CHANGELOG.md` file, moving in the _Release Notes_ that are currently in `README.md`.